src/components/project/query.helpers.ts

```diff
@@ -0,0 +1,46 @@
+import { inArray, node, Query, relation } from 'cypher-query-builder';
+import { ProjectFilters } from './dto';
+
+export function projectListFilter(query: Query, filter: ProjectFilters) {
+  query
+    .match([...propMatch('status')])
+    .call((q) =>
+      filter.status ? q.where({ status: { value: inArray(filter.status) } }) : q
+    )
+    .match([...propMatch('sensitivity')])
```

Comment:
We only want to match these two if they are given

Reply:
You mean, I have to change `match` to `optionalMatch` for these two fields?

Reply:
I mean don't call match if the value is nil

Reply:
Okay